cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/DefaultTaskLocalizationContext.java

```diff
@@ -21,12 +21,13 @@
 
 import java.io.File;
 import java.io.FileNotFoundException;
+import java.io.Serializable;
 import java.util.Map;
 
 /**
  * An implementation of {@link TaskLocalizationContext} that can be initialized with some localized resources.
  */
-public class DefaultTaskLocalizationContext implements TaskLocalizationContext {
+public class DefaultTaskLocalizationContext implements TaskLocalizationContext, Serializable {
```

Comment:
Even if it is the first version of the `Serializable`, would it be better to include `serialVersionUID` field?